app/src/main/java/mozilla/lockbox/view/ItemDetailFragment.kt

```diff
@@ -102,6 +90,20 @@ class ItemDetailFragment : BackableFragment(), ItemDetailView {
         inputLayoutPassword.isHintAnimationEnabled = false
 
         inputUsername.readOnly = true
+        when (showUsernamePlaceholder) {
+            true -> {
+                view!!.btnUsernameCopy.setColorFilter(resources.getColor(R.color.white_60_percent))
+                view!!.isClickable = false
+                view!!.isFocusable = false
+                inputUsername.setText(" ", TextView.BufferType.NORMAL)
```

Comment:
Make this a string resource.

Reply:
The resources aren't accessible from the ItemDetailFragment because it doesn't have any Context